media/js/base/fx-is-default.es6.js

```diff
@@ -21,19 +21,20 @@ const FirefoxDefault = {
         return Mozilla.Client._isFirefoxDesktop() && 'Promise' in window;
```

Comment:
BTW while at it,

```suggestion
        return Mozilla.Client.isFirefoxDesktop && 'Promise' in window;
```
according to: https://github.com/mozilla/bedrock/blob/ea4e454f580c39ea213d1d02b060abcef3b2e074/media/js/base/mozilla-client.js#L533-L535 should be faster.(?)